### PR TITLE
DataFrame.agg() with single non-list aggregation name now returns series

### LINKED ISSUE #235: DataFrame.agg() with single non-list aggregation name returns DataFrame instead of Series
The following code produces incorrect output:

```python
from eland import DataFrame
from eland.tests.common import ES_TEST_CLIENT

df = DataFrame(ES_TEST_CLIENT, "flights")[["FlightDelayMin"]]
print("Eland")
print(df.agg("mean"))
print("Pandas")
print(df.to_pandas().agg("mean"))
```

```
Eland
      FlightDelayMin
mean       47.335171
Pandas
FlightDelayMin    47.335171
dtype: float64
```

### PR DESCRIPTION
**Pull request by V1NAY8**
Closes #235 
- Now non-list aggregation will return series 
- List aggregation works the same.
- Added Tests

### CODE REVIEW

**Comment by V1NAY8** [comment]
@sethmlarson Can you please review this?

File: eland/dataframe.py [hunk 1]
```diff
@@ -1386,9 +1386,12 @@ def aggregate(self, func, axis=0, *args, **kwargs):
         # ['count', 'mad', 'max', 'mean', 'median', 'min', 'mode', 'quantile',
         # 'rank', 'sem', 'skew', 'sum', 'std', 'var', 'nunique']
         if isinstance(func, str):
-            # wrap in list
-            func = [func]
-            return self._query_compiler.aggs(func)
+            if hasattr(self, func):
```
Comment (sethmlarson) [inline eland/dataframe.py:1389]: I think we can do this instead to guarantee that we only run actual aggs like `mean`, etc:

```python
        if isinstance(func, str):
            return self._query_compiler.aggs([func]).squeeze().rename(None)
```

Comment (V1NAY8) [reply eland/dataframe.py]: @sethmlarson That was a brilliant thought of converting dataframe to series. I was just trying to work whole around and making it complex.

I added one more edge case where ed_df.agg(["mean"]) also returns dataframe to series.

Comment (sethmlarson) [reply eland/dataframe.py]: Actually our previous behavior here was correct:

```python
>>> print(df.agg("mean"))
FlightTimeMin     511.127842
FlightTimeHour      8.518797
dtype: float64

>>> print(df.agg(["mean"]))
      FlightTimeMin  FlightTimeHour
mean     511.127842        8.518797
```

If there isn't one already there should be a test case for this behavior.

**Comment by sethmlarson** [comment]
jenkins test this please

**Pull request merged**